bibtexparser/bwriter.py

```diff
@@ -68,8 +68,11 @@ def __init__(self, write_common_strings=False):
         self.display_order = []
         #: BibTeX syntax allows comma first syntax
         #: (common in functional languages), use this to enable
-        #: comma first syntax as the bwritter output
+        #: comma first syntax as the bwriter output
         self.comma_first = False
+        #: BibTeX syntax allows the comma to be optional at the end of the last field in an entry.
+        #: Use this to enable writing this last comma in the bwriter output. Defaults: False.
+        self.comma_last = False
```

Comment:
My 2p: `comma_last` implies a symmetry with `comma_first` although this is a completely different feature. Something like `add_trailing_comma` would be less confusing.

Reply:
`add_trailing_comma` would be fine with me. Maybe I'll wait for input from @sciunto and @omangin before modifying the pull request. Do I need to rebase or submit a different PR?

Reply:
I agree with changing the name. Please do and I will merge just after. (And sorry for the long delay!)